homeassistant/helpers/restore_state.py

```diff
@@ -164,23 +164,20 @@ def async_get_stored_states(self) -> List[StoredState]:
     @callback
     def async_setup_dump(self, *args: Any) -> None:
         """Set up the restore state listeners."""
+
+        def _async_dump_states() -> None:
```

Comment:
This needs to accept an event (time) parameter when it's used as a callback for `async_track_time_interval` and `async_listen_once`.

Reply:
Oops! Good catch.